website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -6752,6 +6861,43 @@ Currently Python external transforms are limited to dependencies available in co
 Go currently does not support creating cross-language transforms, only using cross-language
 transforms from other languages; see more at [BEAM-9923](https://issues.apache.org/jira/browse/BEAM-9923).
 
+#### 13.1.4. Selecting a URN for Cross-language Transforms
+
+Developing a cross-language transform involves defining a URN for registering the transform with an expansion service. In this section
+we provide a convention for defining such URNs. Following this convention is optional but following it will make sure that your transform
```

Comment:
"but following it will make sure" -> "but it will ensure"

Reply:
Done.